sdk/servicebus/azure-messaging-servicebus-track1-perf/src/main/java/com/microsoft/azure/servicebus/perf/App.java

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.azure.servicebus.perf;
+
+import com.azure.perf.test.core.PerfStressProgram;
+
+/**
+ * Runs the Storage performance test.
+ *
+ * <p>To run from command line. Package the project into a jar with dependencies via mvn clean package.
+ * Then run the program via java -jar 'compiled-jar-with-dependencies-path' </p>
+ *
+ * <p> To run from IDE, set all the required environment variables in IntelliJ via Run -&gt; EditConfigurations
+ * section.
+ * Then run the App's main method via IDE.</p>
+ */
+public class App {
+
+    /**
+     *  main function.
+     * @param args args
+     * @throws RuntimeException If not able to load test classes.
+     */
+    public static void main(String[] args) {
+        Class<?>[] testClasses;
+
+        try {
+            testClasses = new Class<?>[]{
+                Class.forName(ReceiveAndDeleteMessageTest.class.getName()),
+                Class.forName(ReceiveAndLockMessageTest.class.getName()),
+                Class.forName(SendMessageTest.class.getName()),
+                Class.forName(SendMessagesTest.class.getName())
+            };
```

Comment:
@g2vinay can we update the guides and all uses of this just to use `.class`?

```java
PerfStressProgram.run(new Class<?>[] {
  ReceiveAndDeleteMessageTest.class,
  ReceiveAndLockMessageTest.class,
  SendMessageTest.class,
  SendMessagesTest.class
}, args);
```

There should be no need to get the `Class` by name as it is already within the scope of the project using it.

Reply:
I think so. I happened to have the same comment 4 lines above.